lib/manager.ts

```diff
@@ -356,6 +356,10 @@ export class Manager implements MCManager {
     }
   }
 
+  fetch(path: string, options?: RequestInit) {
```

Comment:
```suggestion
  fetch(path: string | Request, options?: RequestInit) {
```

let's also the allow a Request object here, like the original fetch?
https://developer.mozilla.org/en-US/docs/Web/API/fetch